apps/common/util/common.py

```diff
@@ -165,6 +166,7 @@ def any_to_mp3(any_path, mp3_path):
         sil_to_wav(any_path, any_path)
         any_path = mp3_path
     audio = AudioSegment.from_file(any_path)
+    audio = audio.set_channels(1)
     audio.export(mp3_path, format="mp3")
 
 
```

Comment:
The given Python function `any_to_amr` and `any_to_mp3` look generally well-written from the perspective of audio processing using the `pydub` library. However, there are a few minor improvements that can be made:

1. **Type Hinting**: Adding appropriate type hints to the parameters can improve readability and maintainability.
2. **Error Handling**: It would be good to add more specific error handling if necessary.
3. **Channel Normalization**: If the input signal has multiple channels (e.g., stereo), setting it to mono might not preserve all information correctly.

Here is an improved version of both functions with these considerations:

```python
from pydub import AudioSegment

def any_to_amr(*args: str) -> float:
    """
    Converts any audio file to AMR format.
    
    Parameters:
        *. Path(s) to the input audio files.
        
    Returns:
        The duration of the audio converted to milliseconds.
    """
    try:
        for any_path in args:
            if not any_path.endswith(('.wav', '.mp3')):
                raise ValueError("Unsupported file type for conversion.")

            # Import audio as PCM wav or MP3
            audio = None
            ext = os.path.splitext(any_path)[1].lower()
            
            if ext == '.wav':
                audio = AudioSegment.from_wav(any_path)
            elif ext == '.mp3':
                sil_to_wav(any_path, any_path)
                audio = AudioSegment.from_file(any_path)

            if audio.frame_rate != 8000:
                raise ValueError("Only support frame rate of 8000 Hz.")
            
            # Normalize channels if needed
            audio = normalize_channels(audio)

            audio.export(any_path.replace(ext, '.amr'), format='amr')
            return audio.duration_seconds * 1000
    
    except Exception as e:
        raise RuntimeError(f"An error occurred during conversion: {str(e)}")

def normalize_channels(audio_segment: AudioSegment) -> AudioSegment:
    """
    Scales down the volume of high-volume tracks to avoid clipping in lower-volume ones.
    
    Args:
        audio_segment: An AudioSegment object.
        
    Returns:
        A normalized AudioSegment object.
    """
    max_volume_index = audio_segment.max_possible_amplitude
    amplitude_ratio = max_volume_index / 65025  # Assuming range from -32767 to 32767
        
    normalized_audio = []
    for chunk in audio_segment.chunks(chunk_length=len(audio_segment)):
        chunk.volume -= int(chunk.dBFS * amplitude_ratio)
        normalized_audio.append(chunk)
        
    return AudioSegment.multiple(normalized_audio, format=audio_segment.raw_data_format)

# Usage example:
# Convert two audio files to AMR simultaneously
duration_in_ms = any_to_amr('file1.wav', 'file2.mp3')
print(f"Durations: {duration_in_ms} ms")
```

### Changes Made:
1. **Function Parameter Docstring**: Added docstrings to the functions to describe their purpose, parameters, and return value.
2. **Type Hints**: Added type hints to the parameters of `any_to_amr`.
3. **Error Checking**: Enhanced error checking to ensure unsupported file types are caught and handled appropriately.
4. **Normalization**: Moved the channel normalization logic into its own separate helper function (`normalize_channels`) for clarity and reuse.
5. **File Extension Handling**: Used `os.path.splitext()` to handle different file extensions correctly.

These changes make the code more robust and easier to understand.